reaktive/src/nativeCommonMain/kotlin/com/badoo/reaktive/utils/CreateDefaultUncaughtErrorHandler.kt

```diff
@@ -1,6 +1,13 @@
 package com.badoo.reaktive.utils
 
+import platform.posix.EXIT_FAILURE
+import platform.posix.fprintf
+import platform.posix.stderr
+import kotlin.system.exitProcess
+
 internal actual fun createDefaultUncaughtErrorHandler(): (Throwable) -> Unit =
     { e ->
-        throw e
+        fprintf(stderr, "Uncaught exception: $e")
```

Comment:
Why both the fprint and the printstacktrace? Looks like it could generate messy stacktraces.

Reply:
`fprintf` just prints the title (exception class name and the message, with red colour), `printStackTrace` prints the same title and the stack trace but in regular colour (looks like via stdout).

Reply:
Ah, of course.